Add user interface to view executed jobs (#104)
yokai-php/batch-src: src/batch-symfony-framework/docs/ui.md

# User Interface

The package is shipped with few routes that will allow you and your users, to watch for `JobExecution`.

<img src="images/bootstrap4-list.png" alt="Bootstrap 4 - List action" width="23%"> <img src="images/bootstrap4-details.png" alt="Bootstrap 4 - Detail : Information" width="23%"> <img src="images/bootstrap4-children.png" alt="Bootstrap 4 - Detail : Children" width="23%"> <img src="images/bootstrap4-warnings.png" alt="Bootstrap 4 - Detail : Warnings" width="23%">

## Installation

> For the UI to be enabled, it is required that you have some optional dependencies:

Reviewer: ```suggestion
For the UI to be enabled, it is required that you install some dependencies:
```
If they are required, they are not optional :wink:

```shell
composer require symfony/translation symfony/twig-bundle
```

## Configuration

> The UI is disabled by default, you must ask for it to be enabled:

Reviewer: ```suggestion
The UI is disabled by default, you must enable it explicitely:
```
